app/controllers/schools/participants_controller.rb

```diff
@@ -12,8 +12,7 @@ class Schools::ParticipantsController < Schools::BaseController
 
   def index
     if FeatureFlag.active?(:change_of_circumstances)
-      @mentor_categories = CocSetParticipantCategories.call(@school_cohort, current_user, ParticipantProfile::Mentor)
-      @ect_categories = CocSetParticipantCategories.call(@school_cohort, current_user, ParticipantProfile::ECT)
+      @mentor_categories, @ect_categories = CocSetParticipantCategories3.call(@school_cohort, current_user)
       @transferring_in = @ect_categories.transferring_in + @mentor_categories.transferring_in
       @transferring_out = @ect_categories.transferring_out + @mentor_categories.transferring_out
       @transferred = @ect_categories.transferred + @mentor_categories.transferred
```

Comment:
Here it feels like it could be made a bit easier to follow if we did something like:

```ruby
  all_categories = CocSetParticipantCategories3.call(@school_cohort, current_user)

  @mentor_categories = all_categories.mentor_categories
  @ect_categories = all_categories.ect_categories
  @transferring_in = all_categories.transferring_in
end
```

Or perhaps those assignments don't even need to happen in the controller, but can be called from `@all_categories` in the view?

Reply:
Good idea. Done.

Reply:
I wonder if we can remove the feature flag :change_of_circumstances. 
It seems it will stay active forever.

Reply:
I'll add that as a tech debt ticket, I'd also like to see the back of it.